index.php

```diff
@@ -56,7 +56,23 @@ function getSipCredentials($cloudUsername, $cloudPassword, $cloudDomain, $isToke
     
     $response = json_decode($response, true);
 
-    // get the sip credentials from the response
+    // Step 5: check if lk is set and is valid
+    if (isset($response['lkhash'])) {
+        $ch = curl_init("https://my.nethesis.it/auth");
```

Comment:
Store the value of the remote auth server as an environment variable, so this can be easily changed.

Reply:
What's the point of changing it?